wok/engine.py

```diff
@@ -2,6 +2,7 @@
 import os
 import sys
 import shutil
+import fnmatch
```

Comment:
It would probably be fine to `from fnmatch import fnmatch`, to shorten other code. But that's not terribly important.